### ISSUE TITLE
Shortcuts do nothing when keyboard focus is on a dropdown menu

### ISSUE THREAD

**Issue by walramb** [hydrusnetwork/hydrus#339]
Whenever my keyboard focus is on a dropdown menu, like selection sort order, pressing shortcuts doesn't do anything. Pressing escape while on a dropdown menu doesn't clear its focus either.

**Comment by thatfuckingbird** [comment]
Is this on Linux? If yes, then it is a limitation of Xorg which applies to all applications, not specific to Hydrus. See https://bugreports.qt.io/browse/QTBUG-10508

**Comment by walramb** [comment]
No, it's on windows 10. Also I mean the dropdown list picker widgets, not the cascading menus.

**Comment by imtbl** [comment]
@thatfuckingbird Think that's a Qt upstream bug?

**Comment by thatfuckingbird** [comment]
I think it is upstream specific behavior but not sure it's a bug. You can have different contexts for shortcuts (application level, window level, specific to a widget, etc.) and maybe when that popup is shown it's considered it's own popup window or something so non-global shortcuts might not work - or something like that. I'm not sure, would need to check the docs and investigate.
